src/Http/ServerRequest.php

```diff
@@ -1134,7 +1136,7 @@ public function parseAccept(): array
      */
     public function acceptLanguage(?string $language = null)
     {
-        $raw = $this->_parseAcceptWithQualifier($this->getHeaderLine('Accept-Language'));
+        $raw = (new ContentTypeNegotiation())->parseAccept($this, 'Accept-Language');
```

Comment:
Perhaps the complete `acceptLanguage()` method should be moved/copied to the `ContentTypeNegotiation` class? It would be nice if all `Accept*` header related detection / juggling is in the new class and `ServerRequest` just has convenience wrapper methods.

`ServeRequest::_acceptHeaderDetector()` could also be updated/moved.